doc/version_history.rst

```diff
@@ -6,6 +6,19 @@
 Version History
 ###############
 
+v0.9.0
+======
+
+* Add `close_stream_writer` function that closes an `asyncio.StreamWriter` and waits for it to close.
+* Update code to wait for stream writers to close.
+
+Requires:
+
+* ts_salobj 5.11 - 6
+* ts_idl 1
```

Comment:
What about ts_idl 2?

Reply:
I'll update the requirements. 1 for salobj 5, 2 for salobj 6.